repository/src/main/java/org/alfresco/repo/action/executer/ScriptActionExecuter.java

```diff
@@ -152,9 +152,8 @@ protected void executeImpl(Action action, NodeRef actionedUponNodeRef)
             {
                 // get the references we need to build the default scripting data-model
                 String userName = this.serviceRegistry.getAuthenticationService().getCurrentUserName();
-                NodeRef personRef = this.personService.getPerson(userName);
-                NodeRef homeSpaceRef = (NodeRef)nodeService.getProperty(personRef, ContentModel.PROP_HOMEFOLDER);
-                
+                NodeRef personRef = this.personService.getPersonOrNull(userName);

```

Comment:
can we add logger here in case of personRef is null?

Reply:
We can but TBH I don't think it will add much value as we are not fetching or using any property from Person object further. Person object is not required for script execution performed during rule action.

Reply:
okay, then it would be fine